.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+#version
+python:
+  - "3.6"
+cache: pip
+
+install:
+  - pip install -r requirements.txt
```

Comment:
please place all watever is to be installed in the requirements txt file , otherwise theres no use of the requirements txtx file